types/share_splitting.go

```diff
@@ -61,43 +61,32 @@ func (msw *MessageShareWriter) Count() int {
 // appendToShares appends raw data as shares.
 // Used for messages.
 func AppendToShares(shares []NamespacedShare, nid namespace.ID, rawData []byte) []NamespacedShare {
-	if len(rawData) <= consts.MsgShareSize {
-		rawShare := append(append(
-			make([]byte, 0, len(nid)+len(rawData)),
-			nid...),
-			rawData...,
-		)
-		paddedShare := zeroPadIfNecessary(rawShare, consts.ShareSize)
-		share := NamespacedShare{paddedShare, nid}
-		shares = append(shares, share)
-	} else { // len(rawData) > MsgShareSize
-		shares = append(shares, splitMessage(rawData, nid)...)
-	}
+	shares = append(shares, splitMessage(rawData, nid)...)
 	return shares
 }
 
-// splitMessage breaks the data in a message into the minimum number of
-// namespaced shares
```

Comment:
Was there a reason to remove documentation?

Reply:
:grimacing: whoops, good catch added back in 1b766ee041186d6e3197caee2403ee5538f5df94